extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -382,7 +391,14 @@ export class Visibility {
       state[LAST_CHANGE_ENTRY] = change;
 
       // Update states and check if all conditions are satisfied
-      if (this.updateCounters_(visible, listener, shouldBeVisible)) {
+      const conditionsMet =
+          this.updateCounters_(visible, listener, shouldBeVisible);
+
+      if (!shouldBeVisible) {
+        // For "hidden" trigger, only update state, don't trigger.
```

Comment:
Yes, saw that. I think I might be understanding this wrong. So just want to make sure that I understand it correctly. What happens for following situation:

"hidden", minContinuousTime = 5 seconds

- Page is loaded, element is on screen for 2 seconds
- Page goes in background
- Does the trigger fire after 3 seconds or after the page comes to foreground again?